### PR TITLE
PR: Ensure Help pane actions are enabled after being undocked

### LINKED ISSUE #11591: Options for different modes of documentation are not available when the Help Pane is undocked.
## Problem Description
 
Options for the pane's menu are deactivated when the pane is undocked

Screen Shot 2020-02-18 at 2 52 19 PM

### What steps reproduce the problem?

1. 
2. 
3. 

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: 

### Dependencies

```
PASTE DEPENDENCIES HERE
```

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Enable all the actions when refreshing them for example when undocking:

help

### Issue(s) Resolved

Fixes #11591 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

### CODE REVIEW

File: spyder/plugins/base.py [hunk 1]
```diff
@@ -406,6 +406,11 @@ def _refresh_actions(self):
 
         # Create actions list
         self._plugin_actions = self.get_plugin_actions() + additional_actions
+        # Ensure all actions are enabled when adding them to the menu.
+        # See spyder-ide/spyder#11591
+        for action in self._plugin_actions:
+            if action is not None:
```
Comment (ccordoba12) [inline spyder/plugins/base.py]: Instead of `None`, perhaps we should use `MENU_SEPARATOR` here?

**Comment by dalthviz** [comment]
Note: I'm quite unsure about the PEP8 error that popped-up (seems like the line has an indentation of 12 spaces / 3 tabs).

File: spyder/plugins/base.py [hunk 2]
```diff
@@ -406,6 +406,11 @@ def _refresh_actions(self):
 
         # Create actions list
         self._plugin_actions = self.get_plugin_actions() + additional_actions
+        # Ensure all actions are enabled when adding them to the menu.
+        # See spyder-ide/spyder#11591
+        for action in self._plugin_actions:
+            if action is not MENU_SEPARATOR:
+                action.setEnabled(True)
```
Comment (ccordoba12) [inline spyder/plugins/base.py]: Not all actions are enabled or should be enabled when refreshing the Options menu. For example, the Options menu of the editor has one disabled action by default:

Selección_114

How does your proposed solution work in that case?

Comment (dalthviz) [reply spyder/plugins/base.py]: I need to rethink the approach here 🤔 (this is more a workaround than an actual fix of the root cause of the problem to be fair)

Comment (dalthviz) [reply spyder/plugins/base.py]: Although testing a little bit seems like this change doesn't affect the current disabled actions. For example, in the Editor the action `Close this panel` remains disable (I think the plugins and widgets change the state of the actions after the `_refresh_actions` is called):

help2

Comment (ccordoba12) [reply spyder/plugins/base.py]: > this is more a workaround than an actual fix of the root cause of the problem to be fair

So what could be the cause of this problem?

Comment (dalthviz) [reply spyder/plugins/base.py]: Maybe that an ancestor is being disable or hide when undocking and the actions get disabled. Anyhow, I need to check more deeply what its actually happening here

**Pull request merged**